addons/dark-www/addon.json

```
@@ -2343,7 +2362,7 @@
         "blue": "#e9f1fc",
         "input": "#fafafa",
         "button": "#4d97ff",
-        "link": "#4d97ff",
+        "link": "#2a77e4",
```

Also note that I darkened the link color for Scratch's default colors (blue).

---

I think users will prefer an unmodified version of the old colors.

---

Note that this line refers to the new `Scratch's default colors` preset (not the "blue" version).

@DNin01 Did Scratch modify the link color for links in the scratch-www repo?

---

> ~Note that this line refers to the new `Scratch's default colors` preset (not the "blue" version).~ Sorry, the line numbers confused me. This does affect the "blue" version of the default colors.
> 
> @DNin01 What's the motivation for changing this preset instead of directly cloning the old preset as of v1.33.0?

I just wanted to increase the contrast a little because the old color probably wouldn't pass any guidelines.

Thinking about it now, this might not be the only element that is inaccessible with this blue color and the user can always change it if necessary, so maybe we shouldn't default to a darker link color in this preset.
```suggestion
        "link": "#4d97ff",
```